modules/cnf-programming-the-sriov-operator.adoc

```diff
@@ -0,0 +1,403 @@
+----
+
+. Find the physical function (PF) of the SR-IOV FEC accelerator device to be configured:
++
+[source,terminal]
+----
+$ oc get sriovfecnodeconfig node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2021-03-19T17:19:37Z"
+      message: Configured successfully
+      observedGeneration: 1
+      reason: ConfigurationSucceeded
+      status: "True"
+      type: Configured
+    inventory:
+        sriovAccelerators:
+        - deviceID: 0d5c
+          driver: ""
+          maxVirtualFunctions: 16
+          pciAddress: 0000.1d.00.0 <1>
+          vendorID: "8086"
+          virtualFunctions: [] <2>
+----
+<1> This field indicates the PCI Address of the card.
+<2> This field shows that the virtual functions are empty.
+
+. Configure the FEC device with the desired setting.
+
+.. Create the following custom resource (CR) and save the YAML in the `sriovfec_n3000_cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: sriovfec.intel.com/v1
+kind: SriovFecClusterConfig
+metadata:
+    name: config
+    namespace: vran-acceleration-operators
+    spec:
+      nodes:
+        - nodeName: node1 <1>
+          physicalFunctions:
+            - pciAddress: 0000.1d.00.0 <2>
+              pfDriver: pci-pf-stub
+              vfDriver: vfio-pci
+              vfAmount: 2 <3>
+              bbDevConfig:
+                n3000:
+                  # Network Type: either "FPGA_5GNR" or "FPGA_LTE"
+                  networkType: "FPGA_5GNR"
+                  pfMode: false
+                  flrTimeout: 610
+                  downlink:
+                    bandwidth: 3
+                    loadBalance: 128
+                    queues: <4>
+                      vf0: 16
+                      vf1: 16
+                      vf2: 0
+                      vf3: 0
+                      vf4: 0
+                      vf5: 0
+                      vf6: 0
+                      vf7: 0
+                  uplink:
+                    bandwidth: 3
+                    loadBalance: 128
+                    queues: <5>
+                      vf0: 16
+                      vf1: 16
+                      vf2: 0
+                      vf3: 0
+                      vf4: 0
+                      vf5: 0
+                      vf6: 0
+                      vf7: 0
+----
+<1> Specify the node name.
+<2> Specify the PCI Address of the card on which the SR-IOV-FEC Operator will be installed.
+<3> Specify the number of virtual functions. Create two virtual functions.
+<4> On `vf0` create one queue with 16 buses (downlink and uplink).
+<5> On `vf1` create one queue with 16 buses (downlink and uplink).
++
+[NOTE]
+====
+For Intel PAC N3000 for vRAN Acceleration the user can create up to 8 VF devices. Each FEC PF device provides a total of 64 queues to be configured, 32 queues for uplink and 32 queues for downlink. The queues would be typically distributed evenly across the VFs.
+====
+.. Apply the CR:
```

Comment:
Please add a blank line above this step.

If you have the time, please let me know which procedure you used as a touchstone--that uses "Apply the CR".  I'll put it on my list of "doc improvements" to make.